src/style.css

```diff
@@ -0,0 +1,120 @@
+:root {
+  --secondary-color: #616070;
+  --buy-link-color: #00acdc;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.card__text {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__add-info,
+.card__rating-feedbacks {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0;
+}
+
+.card__add-info {
+  color: var(--secondary-color);
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  display: flex;
+  align-items: center;
+  /* align-content: center; */
+
+  justify-content: space-between;
+  height: 16px;
+  margin: 0 0 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 16px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-color);
+  margin: 0;
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  margin: 0;
+}
+
+.card__buy-item {
+  width: 100%;
+  height: 40px;
```

Comment:
redundant. you already set line-height to 40px.
and proofread [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)
![Screenshot from 2023-05-13 17-46-42](https://github.com/mate-academy/layout_product-cards/assets/33029864/ff0eb567-d9f9-4c34-a857-97155c9ad57c)

set border for .card__buy-item and change only border-color on hover